README.md

```diff
@@ -86,60 +86,60 @@ $ bash build.sh
+| container list   | export container       | <kbd>e</kbd>                                                   |
+| container list   | commit container       | <kbd>c</kbd>                                                   |
+| container list   | rename container       | <kbd>r</kbd>                                                   |
+| container list   | refresh container list | <kbd>Ctrl</kbd> + <kbd>r</kbd>                                 |
+| container list   | filter image           | <kbd>f</kbd>                                                   |
+| container list   | exec container cmd     | <kbd>Ctrl</kbd> + <kbd>c</kbd>                                 |
+| volume list      | create volume          | <kbd>c</kbd>                                                   |
+| volume list      | remove volume          | <kbd>d</kbd>                                                   |
+| volume list      | prune volume           | <kbd>p</kbd>                                                   |
+| volume list      | inspect volume         | <kbd>Enter</kbd> / <kbd>o</kbd>                                |
+| volume list      | refresh volume list    | <kbd>Ctrl</kbd> + <kbd>r</kbd>                                 |
+| volume list      | filter image           | <kbd>f</kbd>                                                   |
+| network list     | inspect network        | <kbd>Enter</kbd> / <kbd>o</kbd>                                |
+| network list     | remove network         | <kbd>d</kbd>                                                   |
+| pull image       | pull image             | <kbd>Enter</kbd>                                               |
+| pull image       | close panel            | <kbd>Esc</kbd>                                                 |
+| create container | next input box         | <kbd>↓</kbd> / <kbd>Tab</kbd> / <kbd>Ctrl</kbd> + <kbd>n</kbd> |
+| create container | previous input box     | <kbd>↑</kbd> / <kbd>Ctrl</kbd> + <kbd>p</kbd>                  |
+| create container | close panel            | <kbd>Esc</kbd>                                                 |
+| detail           | cursor dwon            | <kbd>j</kbd>                                                   |
+| detail           | cursor up              | <kbd>k</kbd>                                                   |
+| detail           | page dwon              | <kbd>d</kbd>                                                   |
+| detail           | page up                | <kbd>u</kbd>                                                   |
+| search images    | search image           | <kbd>Enter</kbd>                                               |
+| search images    | close panel            | <kbd>Esc</kbd>                                                 |
+| images           | next image             | <kbd>j</kbd> / <kbd>↓</kbd> / <kbd>Ctrl</kbd> + <kbd>n</kbd>   |
```

Comment:
images panel cannot work in this PR.
If you wan to user the key bind, please add following place.

https://github.com/skanehira/docui/blob/master/panel/searchImageResultPanel.go#L81